splinterd/src/node/runnable/mod.rs

```diff
@@ -85,12 +110,12 @@ impl RunnableNode {
                     // Build the `BiomeResourceProvider` to allow the node to access `Biome` endpoints
                     let mut biome_resource_provider =
                         BiomeResourceProvider::new(&*admin_subsystem.store_factory)?;
-                    auth_configs.append(&mut biome_resource_provider.auth_configs);
+                    rest_api =
+                        rest_api.append_auth_configs(&mut biome_resource_provider.auth_configs);
```

Comment:
This should perhaps move to the builder so that we can remove the mut from 'RunnableNodeRestApiVariant::ActixWeb1(mut rest_api)'